CHANGELOG

```diff
@@ -13,7 +13,7 @@ Agenda:
 - Changed API of related objects. All assignments, motions and custom slides
   are now agenda items and can be hidden.
 - Removed mptt.
-- Added attachments to custom slides.
+- Added attachments to agenda items.
```

Comment:
Ah that's not correct. See two lines before: Assignments are now agenda items but they have not attachments. I propose to revert this change.